backend/cmd/migrate/migrations/000013_create_feedback_table.up.sql

```diff
@@ -0,0 +1,17 @@
+CREATE TABLE feedback (
+  -- client_id: unique id generated by the client and used to identify the feedback submission
+  client_id text PRIMARY KEY,
```

Comment:
debated b/w unique & primary key. decided on the latter since it's an identifier of the submission. also debated b/w text and varchar. decided on text since we're not going to set a length requirement on the value